com.unity.render-pipelines.universal/Documentation~/Terrain-Lit-Shader.md

```diff
@@ -0,0 +1,37 @@
+# **Terrain Lit Shader**
+
+The Universal Render Pipeline (URP) uses the Terrain Lit Shader for Unity Terrain. This Shader is a simpler version of the [Lit Shader](lit-Shader.md). A Terrain can use a Terrain Lit Material with up to eight [Terrain Layers](https://docs.unity3d.com/Manual/class-TerrainLayer.html).
+
+![](Images/URPFeatures-TerrainShader.png)
+
+## Creating a Terrain Lit Material
+
+To create a new Terrain Lit Shader Material:
+
+1. Go to your Project window and right-click in the **Assets** folder
+2. Select **Create > Material**. This adds a new Material to your Unity Project’s Asset folder.
+3. Click on the Material to view it in the Inspector.
+4. Click on the **Shader** drop-down and select **URP > Terrain > Lit**.
+
+## Using a Terrain Lit Material
+
+To use a Terrain Lit Material, you must assign it to a Terrain:
+
+1. View the Terrain in the Inspector and click on the gear button on the right side of the Terrain Inspector toolbar to go to the **Terrain Settings** section.
+2. Either drag and drop or use the radio button to assign your Terrain Lit Material to the **Material** property.
+
+![](Images/TerrainLitShader1.png)
+
+## Using the Paint Holes Tool
```

Comment:
This is the link to Paint Holes in the main manual: https://docs.unity3d.com/Manual/terrain-PaintHoles.html